deploy/crds/planetscale_v2_vitesscell_crd.yaml

```diff
@@ -36,6 +36,16 @@ spec:
               items:
                 type: string
               type: array
+            extraVitessFlags:
+              additionalProperties:
+                type: string
+              description: 'ExtraVitessFlags can optionally be used to override default
+                flags set by the operator, or pass additional flags to vttablet from
```

Comment:
only vttablet?

Reply:
This has now been adjusted per @enisoc's suggestion below https://github.com/planetscale/vitess-operator/pull/14/commits/226927d4cabfc3a79db1fcea1f2079d61b4ec302.